android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/splash/SplashScreen.kt

```diff
@@ -0,0 +1,44 @@
+package co.nimblehq.ic.kmm.suv.android.ui.screens.splash
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.ic.kmm.suv.android.R
+import kotlinx.coroutines.delay
+
+@Composable
+fun SplashScreen(onDidLaunch: () -> Unit) {
+    LaunchedEffect(key1 = true) {
```

Comment:
Seems we don't update the value in `LaunchedEffect`, I think we could get rid of `key1  = true`

Reply:
`key1` is required, so I can't remove this one

<img width="576" alt="Screen Shot 2022-09-21 at 12 07 04" src="https://user-images.githubusercontent.com/19943832/191418788-212fc2e1-a3c3-44ea-aa9f-929945de38f7.png">

Reply:
@suho How about replace `key1` with Unit? 😄 

```
LaunchedEffect(Unit)
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/79/commits/f734fc016f53c04d98972caa71f5415472473153